convert2rhel/actions/system_checks/check_firewalld_availability.py

```diff
@@ -0,0 +1,57 @@
+# Copyright(C) 2023 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__metaclass__ = type
+
+import logging
+
+from convert2rhel import actions, systeminfo
+from convert2rhel.systeminfo import system_info
+
+
+logger = logging.getLogger(__name__)
+
+
+class CheckFirewalldAvailability(actions.Action):
+    id = "CHECK_FIREWALLD_AVAILABILITY"
+
+    def run(self):
+        """Error out if the firewalld service is running on the system."""
+        super(CheckFirewalldAvailability, self).run()
+        logger.task("Prepare: Check that firewalld is running")
+
+        if system_info.version.major < 8:
+            description = "Skipping the check as it is relevant only for Oracle/Alma/Rocky Linux 8."
+            logger.info(description)
+            self.add_message(
+                level="INFO",
+                id="CHECK_FIREWALLD_AVAILABILITY_SKIP",
+                title="Skipping the check for firewalld availability.",
+                description=description,
+            )
+            return
```

Comment:
@dkubek @r0x0d does this happen on all RHEL 8 systems or just 8.8? Can we limit it to 8.8 only?

Reply:
I will take a look

Reply:
Looks like it happens only for 8.8. 

Tried with 8.6 conversions in OL and seems that it is not happening. It is probably related to the version rather than the system itself. I will try to investigate this next week.

Reply:
It's related to the packages since updates in ~ mid September '23. Not really system, nor minor version related.
There have been some updates since then, and it seems, that whatever is causing this got fixed for Alma and Rocky Linux.